iceoryx_hoofs/include/iceoryx_hoofs/log/platform/logger.hpp

```diff
@@ -14,11 +14,24 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include "iceoryx_hoofs/internal/log/hoofs_logging.hpp"
+#ifndef IOX_HOOFS_PLATFORM_LOG_LOGGER_HPP
+#define IOX_HOOFS_PLATFORM_LOG_LOGGER_HPP
+
+#include "iceoryx_hoofs/log/platform_building_blocks/console_logger.hpp"
+#include "iceoryx_hoofs/log/platform_building_blocks/logger.hpp"
 
 namespace iox
 {
-constexpr char LoggingComponentHoofs::Ctx[];
-constexpr char LoggingComponentHoofs::Description[];
+namespace platform
```

Comment:
@MatthiasKillat you're understanding is correct.

No, neither `platform` nor `pbb` are mandatory but we should settle on a common namespace. I'm open for suggestions.

Reply:
Sure, I am asking for conventions because of refactoring of error handling. I agree there should be namespaces for it, I am bad coming up with names though but prefer short names here.